odoo/addons/base/res/res.country.state.csv

```diff
@@ -414,43 +414,43 @@ state_it_vr,it,"Verona","VR"
 state_it_vv,it,"Vibo Valentia","VV"
 state_it_vi,it,"Vicenza","VI"
 state_it_vt,it,"Viterbo","VT"
-state_es_c,es,"A Coruña","C"
-state_es_vi,es,"Álava","VI"
+state_es_vi,es,"Araba/Álava","VI"
 state_es_ab,es,"Albacete","AB"
-state_es_a,es,"Alicante","A"
+state_es_a,es,"Alicante / Alacant","A"
 state_es_al,es,"Almería","AL"
 state_es_o,es,"Asturias","O"
 state_es_av,es,"Ávila","AV"
 state_es_ba,es,"Badajoz","BA"
-state_es_pm,es,"Balears","PM"
+state_es_pm,es,"Baleares / Illes Balears","PM"
 state_es_b,es,"Barcelona","B"
 state_es_bu,es,"Burgos","BU"
 state_es_cc,es,"Cáceres","CC"
 state_es_ca,es,"Cádiz","CA"
 state_es_s,es,"Cantabria","S"
-state_es_cs,es,"Castellón","CS"
+state_es_cs,es,"Castellón / Castelló","CS"
 state_es_ce,es,"Ceuta","CE"
 state_es_cr,es,"Ciudad Real","CR"
 state_es_co,es,"Córdoba","CO"
 state_es_cu,es,"Cuenca","CU"
-state_es_gi,es,"Girona","GI"
+state_es_gi,es,"Gerona / Girona","GI"
 state_es_gr,es,"Granada","GR"
 state_es_gu,es,"Guadalajara","GU"
-state_es_ss,es,"Guipúzcoa","SS"
+state_es_ss,es,"Guipúzcoa / Guipuzkoa","SS"
 state_es_h,es,"Huelva","H"
 state_es_hu,es,"Huesca","HU"
 state_es_j,es,"Jaén","J"
+state_es_c,es,"La Coruña / A Coruña","C"
 state_es_lo,es,"La Rioja","LO"
 state_es_gc,es,"Las Palmas","GC"
 state_es_le,es,"León","LE"
-state_es_l,es,"Lleida","L"
+state_es_l,es,"Lérida / Lleida","L"
 state_es_lu,es,"Lugo","LU"
 state_es_m,es,"Madrid","M"
 state_es_ma,es,"Málaga","MA"
-state_es_ml,es,"Melilla","ML"
+state_es_me,es,"Melilla","ME"
```

Comment:
Well, I think we can prepare a migration script for changing it. If you don't update, no problem. If you update, no problem also :wink: If not, we have the same problem with the migration path to v11, but that problem can also be handled in the migration scripts v10 > v11. What you prefer.